sktime/classification/distance_based/_time_series_neighbors_tslearn.py

```diff
@@ -55,6 +55,24 @@ class KNeighborsTimeSeriesClassifierTslearn(_TslearnAdapter, BaseClassifier):
         Above 50, the output is sent to stdout.
         The frequency of the messages increases with the verbosity level.
         If it more than 10, all iterations are reported.
+
+    Examples
+    --------
+    >>> from sktime.classification.distance_based import KNeighborsTimeSeriesClassifierTslearn # doctest: +SKIP # noqa
```

Comment:
This is the main reason CI is failing. I already suggested previously to remove `# noqa` and split this import across multiple linesif flake8/black complains.